management_api.go

```diff
@@ -134,9 +134,14 @@ func (e *Enforcer) AddPolicies(rules [][]string) (bool, error) {
 // Otherwise the function returns true by adding the new rule.
 func (e *Enforcer) AddNamedPolicy(ptype string, params ...interface{}) (bool, error) {
 	if strSlice, ok := params[0].([]string); len(params) == 1 && ok {
-		return e.addPolicy("p", ptype, strSlice)
```

Comment:
Why `addPolicy()` is gone?

Reply:
https://github.com/casbin/casbin/issues/663